architecture/image-controller.md

```diff
@@ -1,28 +1,57 @@
 # Image Controller
 
 # Overview
-Image controller sets up and manages container image repositories for an application's components. This enables greater component isolation within Konflux where each component has its own image repository and secret for pushing images built via Konflux.
+Image controller sets up and manages container image repositories in default registry organization.
```

Comment:
Not clear what `default` means here. Would `in configured quay.io organization` be better?